### PR TITLE
docs: renamed doc sections and titles

### PR DESCRIPTION
**Pull request by neeraj-laad**
Only changed sections and titles

### CODE REVIEW

File: docs/stacks/creating-stacks.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Creating Appsody Stacks"
-section: Stacks
+title: "Creating stacks"
+section: Appsodfy stacks
```
Comment (helenmasters) [inline docs/stacks/creating-stacks.md]: this section name has a typo - Appsodfy

Comment (mhamwala) [reply docs/stacks/creating-stacks.md]: we could probably close this, `creating-stacks.md` will be deleted once my `create-or-modify.md` PR goes in

File: docs/stacks/stacks-overview.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Overview for Appsody Stacks"
+title: "Stacks overview"
 path: /docs/stacks/stack-overview
-section: Stacks
+section: Appsodfy stacks
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: maybe we only need to fix this section name then and the other two can be ignored? this section name has a typo - Appsodfy

Comment (neeraj-laad) [reply docs/stacks/stacks-overview.md]: fixed typo

File: docs/stacks/stacks-overview.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Overview for Appsody Stacks"
+title: "Stacks overview"
 path: /docs/stacks/stack-overview
-section: Stacks
+section: Appsody stacks
```
Comment (mhamwala) [inline docs/stacks/stacks-overview.md]: this should be `Stacks`

File: docs/stacks/modifying-stacks.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Modifying Appsody Stacks"
-section: Stacks
+title: "Modifying stacks"
+section: Appsody stacks
```
Comment (mhamwala) [inline docs/stacks/modifying-stacks.md]: this should be `Stacks`

File: docs/stacks/creating-stacks.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Creating Appsody Stacks"
-section: Stacks
+title: "Creating stacks"
+section: Appsody stacks
```
Comment (mhamwala) [inline docs/stacks/creating-stacks.md]: this should be `Stacks`

**Pull request merged**